src/shell.js

```diff
@@ -235,7 +235,7 @@ if (ENVIRONMENT_IS_NODE) {
   // EXPORT_ES6 + ENVIRONMENT_IS_NODE always requires use of import.meta.url,
   // since there's no way getting the current absolute path of the module when
   // support for that is not available.
-  scriptDirectory = require('url').fileURLToPath(new URL('./', import.meta.url)); // includes trailing slash
+  scriptDirectory = nodePath.dirname(require('url').fileURLToPath(import.meta.url)); // includes trailing slash
```

Comment:
Can you remove the trailing comment (and perhaps add `+ '/'` like 3 lines below).  It seems that `scriptDirectory` is expected to end in a `/` and `path.dirname` does not include one.